scale/job/execution/configuration/configurators.py

```diff
@@ -517,17 +496,13 @@ def _configure_regular_job(config, job_exe, job_type, system_logging_level):
         config.add_to_task('post', mount_volumes={output_mnt_name: output_vol_ro},
                            env_vars={'SYSTEM_LOGGING_LEVEL': system_logging_level})
 
+
         # Configure output directory
-        # TODO: original output dir and command arg replacement can be removed when Scale no longer supports old-style
-        # job types
-        env_vars = {'job_output_dir': SCALE_JOB_EXE_OUTPUT_PATH, 'OUTPUT_DIR': SCALE_JOB_EXE_OUTPUT_PATH}
+        env_vars = {'job_output_dir': SCALE_JOB_EXE_OUTPUT_PATH}
         args = config._get_task_dict('main')['args']
 
-        # TODO: Remove old-style logic for command parameters inject when with v6
-        if not JobInterfaceSunset.is_seed_dict(job_type.manifest):
-            args = JobInterface.replace_command_parameters(args, env_vars)
-        else:
-            args = environment_expansion(env_vars, args, remove_extras=True)
+        args = environment_expansion(env_vars, args)#, remove_extras=True)
```

Comment:
was the remove_extras arg removed? If so we should remove the comment

Reply:
Removed 👍